### PR TITLE
Print out the regenerate command

### PR DESCRIPTION
**Pull request by neubig**
This PR adds an echo to print out the command that is used in the "regenerate" script to generate tests, allowing devs using this script to debug more easily.

### CODE REVIEW

File: tests/integration/regenerate.sh [hunk 1]
```diff
@@ -59,6 +59,13 @@ for ((i = 0; i < num_of_tests; i++)); do
       sleep 1
       rm -rf logs
       rm -rf tests/integration/mock/$agent/$test_name/*
+      echo "echo -e \"/exit\n\" | SANDBOX_TYPE=$SANDBOX_TYPE WORKSPACE_BASE=$WORKSPACE_BASE \
```
Comment (li-boxuan) [inline tests/integration/regenerate.sh:62]: Or we could `set -v` to turn on verbose mode before the next command, and then `set +v` to turn it off again?

Comment (rbren) [reply tests/integration/regenerate.sh:62]: Yeah I think `set +x` does the same--not sure on `set +v`

Comment (li-boxuan) [reply tests/integration/regenerate.sh:62]: Sorry yeah I meant by set +x

Comment (li-boxuan) [reply tests/integration/regenerate.sh:62]: Yeah I think `set -x` is better than `set -v`. The difference is, `-x` mode will replace the variable with its actual value, while `-v` mode will print out the script as it is.

Comment (neubig) [reply tests/integration/regenerate.sh]: Thanks, fixed!

**Review by li-boxuan — APPROVED** [review]
Thanks!

**Pull request merged**